undertow/src/main/resources/schema/wildfly-undertow_3_0.xsd

```diff
@@ -526,7 +526,7 @@
     <xs:complexType name="modClusterType">
         <xs:attribute name="name" use="required" type="xs:string"/>
         <xs:attribute name="management-socket-binding" type="xs:string" use="required"/>
-        <xs:attribute name="advertise-socket-binding" type="xs:string" use="required"/>
+        <xs:attribute name="advertise-socket-binding" type="xs:string" use="optional"/>
```

Comment:
This schema has already been published in WildFly 10.1.0.Final, we can't modify it.